lib/server.js

```diff
@@ -1,9 +1,48 @@
 'use strict';
 
-var sync = require('fh-wfm-sync/lib/server')
-  , config = require('./config');
+var sync = require('fh-wfm-sync/lib/server'),
+  config = require('./config'),
+  shortid = require('shortid');
 
 
 module.exports = function(mediator, app, mbaasApi) {
   sync.init(mediator, mbaasApi, config.datasetId, config.syncOptions);
+
+  mediator.subscribe(config.topicPrefix + config.datasetId + ':create', function(resultToCreate, mediatorTopicIdentifier) {
+    // Add an id field required by the new simple store module to the result object to be created
+    resultToCreate.id = shortid.generate();
+
+    mediator.request(config.topicPrefix + 'data:' + config.datasetId + ':create', resultToCreate, {uid: resultToCreate.id})
```

Comment:
It would be good to have a small utility function to generate the topics. 

It would avoid concatenation errors for a single topic.